rpm/python-oq-engine.spec.inc

```diff
@@ -22,7 +22,7 @@ Patch1: openquake.cfg.patch
 
 Requires(pre): shadow-utils
 
-%define common_deps python numpy scipy python-shapely python-psutil python-mock python-futures h5py python-amqp python-celery python-django python-docutils rabbitmq-server sudo
+%define common_deps sudo systemd python numpy scipy python-shapely python-psutil python-mock python-futures h5py python-amqp python-celery python-django python-docutils python-gunicorn rabbitmq-server
```

Comment:
It was already a requirement: lots of our scripts use sudo to change the user to openquake. I just moved the requirement from the end of the line to the start to have a more logic order in the dependencies